Makefile

```diff
@@ -216,7 +202,6 @@ deploy: | $(DOCKER_SOCK)
 ################################################################################
 .PHONY: clean
 clean:
-	@rm -f Dockerfile*
```

Comment:
We have this line (`@rm -f Dockerfile*`) both in upstream and in openshift repos. It is harmless upstream because there are no Dockerfile-s there. I cannot see why `make clean` must remove our `Dockerfile*` in openshift repo, so I agree with this change. However, I think this change must be out of scope of this rebase. Please correct me if I'm wrong.

Reply:
We will have to create a JIRA and go through the whole process for merging the change, if we do this as separate PR.  I am not sure if that much ceremony is necessary for this small change....

Reply:
Just for records: I was wrong saying that removal of `@rm -f Dockerfile*` must be out of scope. This PR brings those `Dockerfile*` , hence we must update `Makefile` accordingly in this PR too.